src/cljs/athens/views/blocks/core.cljs

```diff
@@ -97,9 +105,15 @@
 (defn block-refs-count-el
   [count uid]
   [:div (stylefy/use-style {:margin-left "1em"
+                            :grid-area "refs"
                             :z-index (:zindex-dropdown style/ZINDICES)
                             :visibility (when-not (pos? count) "hidden")})
-   [buttons/button {:primary true :on-click #(rf/dispatch [:right-sidebar/open-item uid])} count]])
+   [buttons/button {:primary true
+                    :on-click (fn [e]
+                                (doall
+                                  (.. e stopPropagation)
+                                  (rf/dispatch [:right-sidebar/open-item uid])))}
+    count]])
```

Comment:
I tried to cover all the cases where we'd need to catch clicks and stop propagation. Not 100% sure I got them all.